public/app/plugins/panel/canvas/editor/TreeNavigationEditor.tsx

```diff
@@ -0,0 +1,218 @@
+import { css } from '@emotion/css';
+import { Global } from '@emotion/react';
+import Tree from 'rc-tree';
+import React, { Key, PureComponent } from 'react';
+import SVG from 'react-inlinesvg';
+
+import { GrafanaTheme, StandardEditorProps } from '@grafana/data';
+import { config } from '@grafana/runtime/src';
+import { getTheme, stylesFactory } from '@grafana/ui';
+
+import { ElementState } from '../../../../features/canvas/runtime/element';
```

Comment:
```suggestion
import { ElementState } from 'app/features/canvas/runtime/element';
```
kinda nicer than the long relative import path